apps/wallet-mobile/src/TxHistory/ActionsBanner.tsx

```diff
@@ -1,32 +1,35 @@
-import {useNavigation} from '@react-navigation/native'
-import {useSwap} from '@yoroi/swap'
-import React, {ReactNode} from 'react'
-import {useIntl} from 'react-intl'
-import {StyleSheet, TouchableOpacity, View} from 'react-native'
-
-import {Icon, Spacer, Text} from '../components'
-import {useSend} from '../features/Send/common/SendContext'
-import {useSwapForm} from '../features/Swap/common/SwapFormProvider'
-import {actionMessages} from '../i18n/global-messages'
-import {useMetrics} from '../metrics/metricsManager'
-import {TxHistoryRouteNavigation} from '../navigation'
-import {useSelectedWallet} from '../SelectedWallet'
-import {COLORS} from '../theme'
-import {useTokenInfo} from '../yoroi-wallets/hooks'
+import { useNavigation } from '@react-navigation/native'
```

Comment:
Hi @euvini 

Please run the linter before push a commit.

The script is `yarn run lint`

In this way you will identify the format issues like this by file.

There is a plugin for prettier for visual studio code that which locates these issues in the same file. I recommend it 100%